scripts/run_soil_model.R

```diff
@@ -0,0 +1,258 @@
+  lon_farmer <- as.numeric(farm_details$longitude)
+  
+  ################# Weather data pulling
+  
+  weather_data = data.frame(past_temperature=rep(NA,12),future_temperature=rep(NA,12),
+                            past_precipitation=rep(NA,12),future_precipitation=rep(NA,12),
+                            past_pevap=rep(NA,12),future_pevap=rep(NA,12))
+  
+  weather_data[,c("past_temperature", "future_temperature")] <- get_monthly_mean_temperature(lon_farmer,lat_farmer)
+  weather_data[,c("past_precipitation", "future_precipitation")] <- get_monthly_mean_precipitation(lon_farmer,lat_farmer)
+  weather_data[,c("past_pevap", "future_pevap")] <- get_monthly_mean_pevap(lon_farmer,lat_farmer)
+  
+  
+  ################# Pulling calculation factors
+  
+  animal_factors <- read_csv(file.path(modelling_data_loc,"data", "carbon_share_manure.csv")) %>% filter(type=="manure") %>% rename(species=manure_source)
+  agroforestry_factors <- read_csv(file.path(modelling_data_loc,"data", "agroforestry_factors.csv")) 
+  crop_data <- read_csv(file.path(modelling_data_loc,"data", "crop_factors.csv"))#, col_types =  "cdddddddd")
+  pasture_data <- read_csv(file.path(modelling_data_loc,"data", "pasture_factors.csv"))
+  
+  ################# Pulling calculation inputs
+  animal_inputs <- read_csv(file.path(project_loc,project_name,"inputs", "animal_inputs.csv"))
+  agroforestry_inputs <- read_csv(file.path(project_loc,project_name,"inputs", "agroforestry_inputs.csv"))
+  crop_inputs <- read_csv(file.path(project_loc,project_name,"inputs", "crop_inputs.csv"))
+  pasture_inputs <- read_csv(file.path(project_loc,project_name,"inputs", "pasture_inputs.csv"))
+  
+  ################# Calculations per parcel and scenario
+  
+  parcels_Cinput =data.frame(parcel_ID=c(),scenario=c(),agroforestry_Cinput=c(),animal_Cinput=c(),crop_Cinputs=c(),pasture_Cinputs=c())
+  for(parcel in parcels_list){
+    for(scenario in c("current","future","baseline")){
+      parcels_Cinput<-rbind(parcels_Cinput,data.frame(parcel_ID=parcel,
+                                                      scenario=scenario,
+                                                      agroforestry_Cinput=get_monthly_Cinputs_agroforestry(agroforestry_inputs, agroforestry_factors, scenario, parcel),
+                                                      animal_Cinput=get_monthly_Cinputs_animals(animal_inputs, animal_factors, scenario, parcel),
+                                                      crop_Cinputs=get_monthly_Cinputs_crop(crop_inputs, crop_data, scenario, parcel),
+                                                      pasture_Cinputs=get_monthly_Cinputs_pasture(pasture_inputs, pasture_data, scenario, parcel)))
+    }
+  }
+  parcels_Cinput <- parcels_Cinput %>% mutate(tot_Cinputs=agroforestry_Cinput+animal_Cinput+crop_Cinputs+pasture_Cinputs)
+  
+  write.csv(parcels_Cinput,file.path(project_loc,project_name,"results/parcels_Cinput.csv"), row.names = TRUE)
```

Comment:
here better to change it to ...."results", "parcels_Cinput.csv") 
This means that there is never an issue for anyone with the /